test/parallel/test-http-res-write-after-end.js

```diff
@@ -34,8 +34,8 @@ const server = http.Server(common.mustCall(function(req, res) {
   res.end();
 
   const r = res.write('This should raise an error.');
-  // Write after end should return true
-  assert.strictEqual(r, true);
+  // Write after end should return false
+  assert.strictEqual(r, false);
```

Comment:
Please note this. For some reason write after end would return `true`. Which is a bit strange and does not align with `Writable`.